src/ui/public/courier/data_source/admin_doc_source.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @name AdminDocSource
+ */
+
+import _ from 'lodash';
+
+import AbstractDocSourceProvider from './_abstract_doc_source';
+import DocStrategyProvider from '../fetch/strategy/doc_admin';
+import DocRequestProvider from '../fetch/request/doc_admin';
+
+export default function DocSourceFactory(Private) {
+  const AbstractDocSource = Private(AbstractDocSourceProvider);
+  const docStrategy = Private(DocStrategyProvider);
+  const DocRequest = Private(DocRequestProvider);
```

Comment:
is docStrategy a function and DocRequest a class, hence the different capitalizations?  Or should they be consistent?